app/models/user.rb

```diff
@@ -1,7 +1,14 @@
 class User < ActiveRecord::Base
   has_many :rides
+  has_many :passengers
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+
+  def is_passenger?(ride_id)
```

Comment:
Unused method argument - `ride_id`. If it's necessary, use `_` or `_ride_id` as an argument name to indicate that it won't be used. You can also write as `is_passenger?(*)` if you want the method to accept any arguments but don't care about them.<br>Rename `is_passenger?` to `passenger?`.